pkgs/development/libraries/physics/yoda/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, python2Packages, root, makeWrapper, zlib, withRootSupport ? false }:
+{ stdenv, fetchurl, pythonPackages, root, makeWrapper, zlib, withRootSupport ? false }:
```

Comment:
Please do not use `pythonPackages` anymore, unless a version set is used by the caller (so also in `all-packages.nix`).

Reply:
You want me to revert back to `python2Packages`? I thought, the purpose of using `pythonPackages` is to show that a package supports both versions.

Reply:
I will be switching `yoda` to use `python3` by default, but that, probably, can not be done until I get `rivet` version 3 working.